web/crypto_utils.py

```
@@ -0,0 +1,149 @@
+
+import base64
+
+# Handle cryptography import for both runtime and linting
+try:
+    from cryptography.fernet import Fernet
+except ImportError:
+    # For type checking only - the real import will succeed at runtime
+    class Fernet:  # type: ignore
+        """Stub class for cryptography.fernet.Fernet"""
+
+        def __init__(self, key):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)